projects/gloo/pkg/api/converters/kube/header_secret.go

```diff
@@ -32,14 +28,17 @@ func (t *HeaderSecretConverter) FromKubeSecret(ctx context.Context, _ *kubesecre
 	}
 
 	if secret.Type == HeaderSecretType {
-		headerName, hasHeaderName := secret.Data[HeaderName]
-		value, hasValue := secret.Data[Value]
-		if !hasHeaderName || !hasValue {
-			contextutils.LoggerFrom(ctx).Warnw("skipping header secret with missing header-name or value field",
+		if len(secret.Data) == 0 {
+			contextutils.LoggerFrom(ctx).Warnw("skipping header secret with no headers",
 				zap.String("name", secret.Name), zap.String("namespace", secret.Namespace))
 			return nil, nil
 		}
 
+		headers := make(map[string]string, 0)
```

Comment:
nit: maps usually aren't initialized with a len or capacity
```suggestion
		headers := make(map[string]string)
```

Reply:
Fixed.